### ISSUE TITLE
Do you have plan to release the training code?

### ISSUE THREAD

**Issue by arvrschool** [ByteDance-Seed/Depth-Anything-3#35]
Great work! Do you have plan to release the training code? Thx~

**Comment by Laurenz-Ruzicka** [comment]
I would also be interested, as this would be very beneficial and helpful

**Comment by hiepbk** [comment]
This is an amazing paper, 
We're waiting for the trainning code to be opened.
That would be great

**Comment by anhquancao** [comment]
Not one of the authors, but since the official training code isn't out yet. We implemented the full training pipeline and losses for the DA3 backbone following the details in the paper. It's in our repo:

https://github.com/valeoai/OccAny#occany-depth-anything-3--sam3-1

- complete training loop with all losses implemented
- trains the DA3 backbone end-to-end, incl. the 1.1B Giant
- SLURM + shell launchers, tested on 16×A100 40GB
